src/components/GlobalPreferences/HelpAndFeedback/HelpAndFeedback.js

```diff
@@ -5,30 +5,29 @@ import helpAndFeedbackSvg from './help-and-feedback.svg'
 
 function HelpAndFeedback({ optedOut, onOptOutChange }) {
   const theme = useTheme()
-  // checked => not opted out
-  const handleOptOutChange = checked => onOptOutChange(!checked)
+  const handleOptOutChange = () => onOptOutChange(!optedOut)
 
   return (
     <Box heading="Help Scout">
-      <label
+      <div
         css={`
-          cursor: pointer;
           display: flex;
-          margin-bottom: ${4 * GU}px;
-          align-items: center;
           justify-content: center;
+          margin-bottom: ${4 * GU}px;
         `}
       >
-        <Checkbox onChange={handleOptOutChange} checked={!optedOut} />
-        <span
-          css={`
-            color: ${theme.surfaceContentSecondary};
-            font-size: 20px;
-          `}
-        >
-          Allow Help Scout feedback module
-        </span>
-      </label>
+        <label css="cursor: pointer">
+          <Checkbox onChange={handleOptOutChange} checked={!optedOut} />
+          <span
+            css={`
+              color: ${theme.surfaceContentSecondary};
+              font-size: 20px;
```

Comment:
Do we have a `textStyle` for `20px`?

Reply:
yes https://github.com/aragon/aragon-ui/blob/newstyle/src/style/text-styles.js#L18

Reply:
Added in 72e17d5.